BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift

```diff
@@ -46,6 +47,7 @@ final class EditLineItemView: UIView {
                                   for: .normal)
         button.addTarget(self, action: #selector(didTapSave), for: .touchUpInside)
         button.setTitleColor(.GiniBank.accent1, for: .normal)
+        button.titleLabel?.textColor = .GiniBank.accent1
```

Comment:
[Clarification] I don't think we need this change.
We have set button.titleLabel?.textColor in line 49

Reply:
Same